.github/workflows/ci.yml

```diff
@@ -25,21 +25,12 @@ jobs:
     - uses: actions/checkout@v3
```

Comment:
do we want to remove line 19 for the python versions? seems redundant?

Reply:
As far as I know, GitHub Actions can only use one python version for each session, so we still need to use matrix to check for multiple python versions. I think it should not perform same test multiple times because we can specify what test to run on each python version at `[gh-actions]` in `tox.ini` using `tox-gh-actions`.